### PR TITLE
UIUX - Centered social media using Bootstrap logic

### PR DESCRIPTION
**Pull request by Siphon880gh**
I added more Bootstrap to center the footer icons and text in mobile. You may notice some floating logic are removed in index.css for social media icons and copyright text because they are no longer necessary once we just leverage Bootstrap for this purpose.

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -7,10 +7,12 @@ import Image from 'react-bootstrap/Image';
...
 	render(){
 		return (
-			<Row>
-				 <Col xs={12} sm={12} md={12} lg={12} xl={12}>
-					<div id="footer">
-						<div id="footer_social_links_container">
+				<footer id="footer">
+					<Row>
+						<Col xs={12} sm={12} md={6} className="text-center text-md-start order-last order-md-first mt-5px" id="copyright_links">
+							© 2021 xSurge. All rights reserved.
```
Comment (Siphon880gh) [inline src/components/Footer.js:13]: Explanation of the logic: So it centers on mobile first. But on medium screens, it's left aligned (text-md-start). We flip the the order of elements between text and social media icons, but flip the order the other way around on mobile. That's order-last vs order-md-first

File: src/components/Footer.js [hunk 2]
```diff
@@ -7,10 +7,12 @@ import Image from 'react-bootstrap/Image';
...
+						</Col>
+						<Col xs={12} sm={12} md={6} className="text-center text-md-end" id="footer_social_links_container">
```
Comment (Siphon880gh) [inline src/components/Footer.js:15]: Explanation of the logic: So it centers on mobile first. But on medium screens, it's right aligned (text-md-end).

File: src/components/Footer.js [hunk 3]
```diff
@@ -29,23 +31,25 @@ class Footer extends Component{
...
-				</Col>
-			</Row>
+						</Col>
+						</Row>
+						
+						{/* <div class="clear"></div> */}
+
+						<Row>
+						<Col xs={12} sm={12} md={12} lg={12} xl={12}>
+							<div class="text-center text-md-start mt-5px" id="disclosure_text">
```
Comment (Siphon880gh) [inline src/components/Footer.js:41]: Explanation of the logic: So it centers on mobile first. But on medium screens, it's left aligned (text-md-start).

**Review by milorddev — APPROVED** [review]

**Review by birthdaysmoothie22 — COMMENTED** [review]
Looks good

**Pull request merged**